packages/ui-library/src/components/counter/index.ts

```diff
@@ -10,26 +10,26 @@ export class BlrCounter extends LitElement {
   static styles = [];
 
   @property() variant: CounterVariantType = 'neutral';
-  @property() current = 0;
-  @property() max = 0;
-  @property() size?: FormSizesType = 'md';
+  @property({ type: Number }) value = 0;
+  @property({ type: Number }) maxValue = 0;
+  @property() sizeVariant?: FormSizesType = 'md';
   @property() theme: ThemeType = 'Light';
 
   protected render() {
-    if (this.size) {
+    if (this.sizeVariant) {
       const dynamicStyles = this.theme === 'Light' ? [counterLight] : [counterDark];
 
       const classes = classMap({
         'blr-counter': true,
         [this.variant]: this.variant,
-        [`${this.size}`]: this.size,
+        [`${this.sizeVariant}`]: this.sizeVariant,
```

Comment:
For consistency, does this need to be in template literal, can't it be just like in line number 24?